metadata-ingestion/src/datahub/ingestion/source/fivetran/config.py

```diff
@@ -167,6 +167,10 @@ class PlatformDetail(ConfigModel):
         description="The database that all assets produced by this connector belong to. "
         "For destinations, this defaults to the fivetran log config's database.",
     )
+    include_schema: bool = pydantic.Field(
+        default=True,
+        description="Include schema in the dataset URN. In some cases, the schema is not relevant to the dataset URN and Fivetran sets it to the connector name.",
+    )
```

Comment:
`include_schema` might be too generic and could lead to misunderstandings. WDYT about something like `include_schema_in_urn` instead?

I'm also a bit confused by the description: 'Include schema in the dataset URN' followed by 'Fivetran sets it to the connector name.' Is it referring to the dataset or the connector?